dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -58,6 +58,28 @@
   = f.select :video_key, options_for_select(video_key_choices, @level.video_key), {include_blank: true}, {class: 'video-dropdown'}
   .video-preview{style: 'display: block'}
 
+.field
+  = f.label :map_levels, 'Map Level'
+  %p If there is a key map level you want to reference in the instructions, add the link here. It will show up under the 'Help & Tips' tab.
+  %p This should be a string representing the URL on docs.code.org you want to embed, i.e. '/cs/maker_leds/index.html'
+  - @level.map_levels = [''] unless @level.map_levels.try(:present?)
```

Comment:
This means if we load the editor, then doing nothing, `map_levels` will start out as nil, and finish as `['']`. Are those two situations equivalent?

Reply:
Is the goal here to reference a map level (i.e. something like https://studio.code.org/s/allthethings/stage/35/puzzle/2), or is it to reference the content that the mapped level links to (https://studio.code.org/docs/applab/checkbox/index.html)?

Reply:
The goal is to reference the content in the mapped level, which will eventually be accessible in a modal from 'this' level.

Reply:
If the goal is to reference the content, it seems like instead of referencing Map Levels, we should just reference the content.

This might partially be a confusion over semantics. Map levels are levels in our system (i.e. they have a /s/scriptname/stage/stagenumber/puzzle/puzzlenumber URL. Essentially all they consist of is a reference to curriculum builder content (either docs.code.org or curriculum.code.org). We then proxy that content through a studio endpoint (either studio.code.org/docs or studio.code.org/curriculum).

I think what we want here is that instead of having a reference to the level (which then has a reference to content), we just want a reference to content.

Reply:
After looking into this further, I'm actually not sure my previous statement was correct, as that would make the Map Levels/Reference Links serve the same purpose. I'm going to touch base with Poorva on this to double check.

Reply:
Alright, looping back. We do want to display them differently so that we can give them different icons, but both types of resource will be displayed the same - as content in a modal.